en/docs/develop/creating-artifacts/creating-endpoints.md

```diff
@@ -46,11 +49,11 @@ The endpoint will be created in the `<PROJECT_NAME>/src/main/wso2mi/artifacts/en
 
 The endpoint artifact is now linked to your integration sequence.
 
-### Updating the properties
```

Comment:
In the **Call Endpoint** form, select the endpoint that you need to invoke under **Select Endpoint**. You can also select **Create New** to create a new endpoint artifact.

->

In the **Call Endpoint** form, select the endpoint you need to invoke under **Select Endpoint**. You can also select **Create New** to create a new endpoint artifact.

Reply:
Fixed in 0cf82a747682ffba5d952c65b402a54950a089d9.